netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -96,6 +96,7 @@
  * the context of the Netty Channel thread.
  */
 class NettyServerHandler extends AbstractNettyHandler {
+
```

Comment:
Please avoid full-file formatters; every developer can have slightly different settings and they can constantly "fight" causing lots of useless churn. Please revert these unnecessary changes.

Reply:
Will do. I'd suggest updating `CONTRIBUTING.md` with that info, I just followed it and used the recommended Google settings for IntelliJ.

Reply:
My bad, `CONTRIBUTING.md` already has it. I missed it somehow:

>>>
Don't fix code style and formatting unless you are already changing that line to address an issue. PRs with irrelevant changes won't be merged. If you do want to fix formatting or style, do that in a separate PR.